packages/react-cookie/src/withCookies.tsx

```diff
@@ -37,35 +37,43 @@ export default function withCookies<T>(WrapperComponent: React.ComponentType<T &
       }
     }
 
+    componentDidMount() {
+      this.listen(this.props.cookies);
```

Comment:
Now that you moved the `cookies` in the props, it no longer needs a private variable holding it. You can remove it from the class, the listen parameter and everywhere using it.

For the check if the `cookies` instance changed, you can move it within `componentDidUpdate` and use the `prevProps` parameter.

Great job overall 👍

Reply:
Thanks! I updated it with the changes you suggested